app/apphandlers/processorAppHandler.go

```diff
@@ -272,8 +273,7 @@ func startHealthWebHandler(ctx context.Context) error {
 	// Port where Processor health handler is running
 	pkgLogger.Infof("Starting in %d", webPort)
 	srvMux := mux.NewRouter()
-	srvMux.HandleFunc("/health", healthHandler)
-	srvMux.HandleFunc("/", healthHandler)
```

Comment:
I am not 100% sure but I guess we use this end point at few places too for alerting. Can we not remove this ?